kiliautoml/utils/helpers.py

```diff
@@ -149,6 +150,30 @@ def get_assets(
     return assets
 
 
+def get_label(asset: AssetT, strategy: LabelMergeStrategyT):
+    TYPE_ORDER = {
```

Comment:
should be generated once and for all, so you should move it outside of the get_label_function (bellow the imports)

Reply:
(for perf reasons)